### DISCUSSION TITLE
Upcoming Webinar 01/11/2022

### DISCUSSION THREAD

**Discussion by shaselton-usds** [CMSgov/price-transparency-guide#322] [General]
Please join the Centers for Medicare &amp; Medicaid Services (CMS) on Tuesday, January 11th 1:00-2:00pm ET for the fourth in a series of technical implementation webinars for the in-network negotiated rate and out-of-network historical billed charges machine-readable file requirements in the Transparency in Coverage (TiC) Final Rules (85 FR 72158).

During this fourth webinar, the CMS team will be sure to recap December’s webinar and cover new decisions arrived on GitHub. CMS will also respond to appropriate questions that are posted in this announcement as well as in the Q&amp;A chat box during the webinar.

Please register for the webinar [here](https://cms.zoomgov.com/webinar/register/WN_itID9daxSFud7hn4jsil8Q).

Post questions below and like/upvote questions so the team knows what’s most important to you.

**Comment by WillFox123** [comment]
1)  How should in-network percent of charge fees be submitted?  This can be a large portion of hospital outpatient claims.    2)  How should Inpatient facility outlier provisions be submitted?  This can have a significant impact on hospital allowable charges.

**Comment by hhCambia** [comment]
Can CMS provide specific explanation as to why the reference file idea expressed in discussion thread #231 should not be adopted?

**Reply by shaselton-usds** [reply]
Hopefully the [recent updates](https://github.com/CMSgov/price-transparency-guide/blob/release/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L25-L28) that allow for external provider referencing in the `provider_references` attribute address this suggestion.

**Comment by skochin** [comment]
Can a Type 2 NPI be used instead of an EIN for a provider groups' "tin"?  Some contracts are written at the billing NPI level, and there may be several under a single business entity (EIN).

**Reply by skochin** [reply]
@shaselton-usds please give us some guidance on this - The schema as currently defined uses "ein" as the primary identifier for a provider group, unless the provider is operating under their personal SSN.  Our plan's contracts define a group of providers under a Type 2 NPI, and more than one could have the same EIN.  If we group providers by EIN, more than one rate contract could apply.  Should we:

1. Keep the groups distinct, despite having the same TIN
  `{
"provider_group_id": 10001,
"provider_group": {
    "npi": [
        1111111111,
        2222222222,
    ],
    "tin": {
        "type": "ein",
        "value": "123456789"
    }
}
},
{
"provider_group_id": 10002,
"provider_group": {
    "npi": [
        3333333333,
        4444444444
    ],
    "tin": {
        "type": "ein",
        "value": "123456789"
    }
}`
2. Combine into one group, but show multiple in-network rates for the same service (this seems to be the most confusing option)
3. Substitute the T2 NPI for the EIN
  `{
"provider_group_id": 10001,
"provider_group": {
    "npi": [
        1111111111,
        2222222222,
    ],
    "tin": {
        "type": "npi2",
        "value": "8888888888"
    }
}
},
{
"provider_group_id": 10002,
"provider_group": {
    "npi": [
        3333333333,
        4444444444
    ],
    "tin": {
        "type": "npi2",
        "value": "9999999999"
    }
}`

**Comment by lkabi** [comment]
What is Provider References? Is it to be used when a Provider Group (based on TaxID) is duplicated?

Can CMS provide an updated example for both INN and ONN MRF layout. And Is the layout order to be followed as per schema? The example provided currently does not match to the INN & ONN schema. 
For example in INN Schema, 'in_network' (array) comes before 'provider_references'. But in example provided for **in-network-rates-fee-for-service-single-plan-sample.json**, 'provider_reference' comes before 'in_network' array.
The same is applicable for 'last_updated-on' & 'version' .Please clarify

**Reply by shaselton-usds** [reply]
This is pretty core to understanding JSON (and XML and various other formats): JSON is [an object of an unordered set of name/value pairs](https://www.json.org/json-en.html). "Unordered" being the keyword here.

As I can tell, assuming that nothing is fundamentally broken, the tests pass for the `in-network-rates-fee-for-service-single-plan-sample.json` file against the schema found [here](https://github.com/CMSgov/price-transparency-guide/blob/release/tests/spec/libs/in-network-rates/in-network-rates_spec.rb#L13-L17)

You could leverage `provider_references` if you are finding that your files are becoming too large when defining the provider groups inline (and repetitively) for all of the items/services that are being reported upon. With `provider_references`, you would define your `provider_groups` once and then "reference" them later in the [`negotiated_rate` object](https://github.com/CMSgov/price-transparency-guide/blob/release/examples/in-network-rates/in-network-rates-fee-for-service-single-plan-sample.json#L87).

**Comment by tdfow** [comment]
What is the expected cadence we should expect once a Pull Request is closed but remains in the develop branch before it is moved to the master branch?   For example, the develop branch has many changes not reflected in the master---so how long will changes remain in 'develop' branch before they are 'official' CMS schema changes?

**Reply by shaselton-usds** [reply]
@tdfow I tried to answer this during the Q&A section of the webinar. The idea at this point is to follow a loose git-flow workflow. There is currently a release branch PR (https://github.com/CMSgov/price-transparency-guide/pull/353) into the master branch that should be merged quickly.

**Comment by brbucher** [comment]
Why is it no longer an option to include the reporting plans object within the files?
Since the allowed amounts and in network rates files will be coming from different sources and in some cases be generated by separate entities, implementing the table of contents file introduces dependencies between the two other files that otherwise don't exist.
To solve this dependency issue, would either of the following options be possible?
A) Reintroduce the option to use the reporting plans object within the files.
Or
B) Separate the table of contents file for the allowed amounts and in network rates files so there would be a table of contents for each.

**Comment by brbucher** [comment]
Would it be possible to add a group of provider groups object for the in network rates object?
From our testing with the provider group references, we are seeing the same group of provider groups appear for multiple codes.
For example: We have provider groups with indices 1,2,3, and 4.
For each billing code, provider groups 1,2, and 3 have the same negotiated price and appear together.
If we were able to group the provider groups, that would reduce the amount of data reported in the example case from three values to one for each billing code.
Especially considering that these groups of provider groups can get very large, we are seeing a group of ~30,000 as the largest, being able to compile a list of those groups early in the file rather than repeating the values for each code could yield a significant improvement to file size.

**Reply by shaselton-usds** [reply]
@brbucher thanks for the heads up -- this was actually not intended. The `provider_groups` within the `provider_references` array was meant to match how `provider_groups` are used in-line in the `negotiated_price` object.

This commit (https://github.com/CMSgov/price-transparency-guide/commit/b68b1c241d6aa52a76c4f590010450f6a6968d9a#diff-6ad0e4105a9c63ddad64bf5d73885fb13e57988d0704f5a00db59d8f94147836) hopefully addressed what you were asking about. Please let me know otherwise.

**Comment by MaIvonneS** [comment]
For version 0.8.0 (introduction of Table of Contents), the reporting_plan_object_types array got removed from the In-Network and OON Allowed Amounts machine readable files schema.  Instead, the 4 header fields (plan_name, plan_id_type, plan_id and plan_market_type) are now string fields again.  Since the Table of Contents is just optional, can we reinstate the reporting_plan_object types array in the final v1.0.0 so there will be an ability to add multiple plans to a single file when negotiated rates and allowed amounts are the same?

**Reply by shaselton-usds** [reply]
@MaIvonneS The `reporting_plan_object` was removed from the in-network and allowed amount file because of the file naming issues that were introduced. Moving that object into the table of contents solved this problem while still allowing for multiple plans to use the same files. 

You are still able to add multiple plans to a single file when the rate are the same within the table of content file. Conceptually, the reporting_plan_object was just moved from within the in-network/allowed amounts files to the table of contents.

**Comment by tdfow** [comment]
@shaselton-usds when is the next webinar?    Will some of the discussion items be addressed?

**Reply by jltang-kp** [reply]
I believe @shaselton-usds mentioned that there will be some direction on percent of billed. Would like to know if this is still up for a topic for the next webinar

**Reply by tdfow** [reply]
@jltang-kp -- I hope so---  I haven't seen any updates to some discussion topics in a bit from CMS, so wondering if those would be addressed in the next webinar.    For example, I believe there is an issue with the current placement of the claim modifier in the Allowed Amount Schema so hoping someone can address this sooner than later.

**Reply by angelamarie0975** [reply]
@shaselton-usds - we are wondering the same thing as well about the next webinar and the items to be discussed. 
1. Percentage of Billed - We are currently blocked for percentage of billed rates as we do not have the ability to get the billed charge in order to calculate the percentage of billed into a dollar amount. In a previous discussion - @tdfow mentioned they were basing the rates off of a dollar as the billed charge in order to calculate to a dollar amount, which we may have to do - however, we would need to be able to add some type of notation that this is a percentage of billed at 1 dollar so that it would make sense to the consumer of these files. 
2. Modifier code for OON - we had this question as well, so we are interested in an answer to that also.
3. Reporting_Entity_Type on both Table of Contents and INN file. I just posted a discussion here - https://github.com/CMSgov/price-transparency-guide/discussions/370

**Reply by shaselton-usds** [reply]
@tdfow We are trying to nail down when that next webinar will be -- we are definitely cognizant of the timeframe.

Hopeful topics will be the percentage of billing along with possible other alternative arrangements, but we also have to understand that this dance of policy and technology happen at different paces. While technology solutions may be determined fairly rapidly, we are trying our best to sync the two.

There is also the possibility of having a webinar where it is just straight Q&A, but we unsure if it would be worth the time because the assumption is that most of the questions will be surround guidance that is still under consideration.

Once the next webinar date is confirmed, we'll definitely post that here as always.
